umbraco-cloud/SUMMARY.md

```diff
@@ -51,7 +51,7 @@
 * [Media](set-up/media/README.md)
   * [Connect to Azure Storage Explorer to upload files manually](set-up/media/connect-to-azure-storage-explorer.md)
 * [Users](set-up/users-on-cloud.md)
-* [2-factor authentication](set-up/2-factor-authentication-on-cloud.md)
+* [Multi-Factor Authentication](set-up/2-factor-authentication-on-cloud.md)
```

Comment:
Should we consider updating the URL as well? 🤔

Reply:
Yup, will rename the file and update the URLs